kuma/wiki/jobs.py

```diff
@@ -67,6 +68,11 @@ class DocumentContributorsJob(KumaJob):
     # Don't synchronously fetch the contributor bar but schedule a fetch
     fetch_on_miss = False
 
+    def get(self, *args, **kwargs):
+        if settings.MAINTENANCE_MODE:
+            return self.empty()
+        return super(DocumentContributorsJob, self).get(*args, **kwargs)
```

Comment:
👍 works locally, good use of ``empty``.  In the sample DB, it can be hard to distinguish the empty contributor list, since all the gravatars are misses - need to find edits by me or sheppy.  We need to fix the default gravitar image in dev...